src/frontend/admin/src/pages/admin/home/index.tsx

```diff
@@ -0,0 +1,64 @@
+import { defineMessages, useIntl } from "react-intl";
+import Grid2 from "@mui/material/Unstable_Grid2";
+import Alert from "@mui/material/Alert";
+import * as React from "react";
+import { useMemo } from "react";
+import { DashboardLayoutPage } from "@/layouts/dashboard/page/DashboardLayoutPage";
+import { LinkCard } from "@/components/presentational/card/LinkCard";
+import { getHomeShortcuts } from "@/pages/admin/home/shortcuts";
+
+const messages = defineMessages({
+  pageTitle: {
+    id: "pages.admin.home.pageTitle",
+    defaultMessage: "Welcome",
+    description: "Label for the home page title",
+  },
+  alertInfo: {
+    id: "pages.admin.home.alertInfo",
+    defaultMessage:
+      "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nunc rutrum, nunc in tempor lacinia, quam eros condimentum nisi, vel sodales massa nisl vel odio.",
+    description: "Text for the alert info in home page",
+  },
+  rootBreadcrumb: {
+    id: "pages.admin.home.rootBreadcrumb",
+    defaultMessage: "Home page",
+    description: "Label for the root breadcrumn ",
+  },
+});
+
+export default function Index() {
+  const intl = useIntl();
+
+  const cards = useMemo(() => getHomeShortcuts(intl), []);
```

Comment:
The naming is weird. The variable is called `card` but the method which should return card data is called `getHomeShortcuts`. Can you just homogenize this plz.

From my point of view, something like `getHomeCardData` is clearer.